### ISSUE TITLE
600,000 iterations

### ISSUE THREAD

**Issue by tgoncalv** [BossunWang/DRB-GAN#1]
My question may be very trivial, but I struggle to understand how we can use 600,000 iterations with a batch size of 1 when the training dataset is supposed to contain 624,777 content images. Unless I have done a huge logical mistake, I'd dare to say that we don't ever obtain one full epoch. Have I missed something crucial in my analysis ?

**Comment by BossunWang** [comment]
I think you can't consider about full dataset as one epoch, training by every iteration only until the loss arrived balance.

**Comment by tgoncalv** [comment]
Well yes, I agree with you. If the loss converges, we can stop the training. But just to be sure, at the end, the generator is trained on only 600,000 images, right ? I'm confused by the specific number "624,777" because I couldn't understand where this value comes from. I tried to check the references and the description of Place365 dataset (I'm not an expert on this dataset) but I couldn't find anything that satisfies me, so why this precise number ?

**Comment by BossunWang** [comment]
I think the size of dataset  not so sensitive, you should ask the author if your goal is experiment setting and compared with the benchmark , I'm not the author.
